contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -70,6 +70,50 @@ reasonable production worthy environments:
   bypass policies that are not yet GA, but which may reasonably be enabled on a
   conformant cluster (e.g., pod security policy, non-GA scheduler annotations)
 
+### Windows & Linux Considerations
+
+As of v1.14, Windows is a stable feature, but not yet included in conformance
+testing. In preparation for this, a large number of conformance tests are
+already included in Windows testing. You can see what tests have already pass by
+looking at TestGrid for results of Windows tests running on 
+[Azure](https://testgrid.k8s.io/sig-windows#aks-engine-azure-windows-master) and 
+[GCE](https://testgrid.k8s.io/sig-windows#gce-windows-master)).
```

Comment:
Add steps to launch these tests, see https://github.com/kubernetes/test-infra/pull/12010

Reply:
Done